pkg/steps/release/create_release.go

```diff
@@ -172,6 +172,12 @@ func (s *assembleReleaseStep) run(ctx context.Context) error {
 	now := time.Now().UTC().Truncate(time.Second)
 	version := fmt.Sprintf("%s-%s-test-%s-%s", prefix, now.Format("2006-01-02-150405"), s.jobSpec.Namespace(), s.name)
 
+	referencePolicy := ""
+	if s.config.ReferencePolicy != "" && s.config.ReferencePolicy == "Source" {
```

Comment:
It would be better to use `imagev1.SourceTagReferencePolicy` rather than the hardcoded `Source..